MAUI/Cartesian-Charts/Legend.md

```diff
@@ -104,6 +104,47 @@ this.Content=chart;
 
 {% endtabs %}
 
+## Customizing labels
+
+The [`Label`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html#Syncfusion_Maui_Charts_CartesianSeries_Label) property of [`CartesianSeries`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Charts.CartesianSeries.html) is used to define the label for the corresponding series legend item. The appearance of the label can be customized using the [`LabelStyle`]() property. 
+
+* [`TextColor`]() – used to change the color of the label.
```

Comment:
Update the definition like this " Gets or sets the color for the text of the label."

Reference - https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#label-customization

Reply:
updated definitions